### PR TITLE
frontend/fix/#1250: bug fixes for nammayatri

### PR DESCRIPTION
**Pull request by ashkriti**
## Type of Change

- [x] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

## Description
Fixed
ID_03 : Current location is not updating properly on start location
ID_04 : Tips pop up close icon is not displayed
ID_08 : User and driver both getting current location as source across the application, from start to till download invocie

## Motivation and Context
https://github.com/nammayatri/nammayatri/issues/1250

## How did you test it?
Tested Manually

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [x] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable

### CODE REVIEW

File: Frontend/android-native/app/src/user/nammaYatri/assets/juspay/juspay_assets.json [hunk 1]
```diff
@@ -129,6 +129,7 @@
       "ny_ic_day" : true,
       "ny_ic_dest_marker" : true,
       "ny_ic_direct_call" : true,
+      "ny_ic_dismiss" : true,
```
Comment (ashishsingh101) [inline Frontend/android-native/app/src/user/nammaYatri/assets/juspay/juspay_assets.json:132]: can you add the image for this

Comment (ashkriti) [reply Frontend/android-native/app/src/user/nammaYatri/assets/juspay/juspay_assets.json:132]: The image has already been added for this in asset store and android as well as in IOS and PR was merged only this JSON had to be pushed

Comment (ashishsingh101) [reply Frontend/android-native/app/src/user/nammaYatri/assets/juspay/juspay_assets.json:132]: okay

Comment (ashishsingh101) [reply Frontend/android-native/app/src/user/nammaYatri/assets/juspay/juspay_assets.json:132]: okay

File: Frontend/ui-customer/src/Flow.purs [hunk 2]
```diff
@@ -1129,10 +1129,17 @@ rideSearchFlow flowType = do
...
-          void $ pure $ setFlowStatusData (FlowStatusData { source : {lat : finalState.props.sourceLat, lng : finalState.props.sourceLong, place : finalState.data.source}
+          (GetPlaceNameResp locationName) <- Remote.placeNameBT (Remote.makePlaceNameReq finalState.props.sourceLat finalState.props.sourceLong (case (getValueToLocalStore LANGUAGE_KEY) of
+                                                                                                                            "HI_IN" -> "HINDI"
+                                                                                                                            "KN_IN" -> "KANNADA"
+                                                                                                                            "BN_IN" -> "BENGALI"
+                                                                                                                            "ML_IN" -> "MALAYALAM"
+                                                                                                                            _      -> "ENGLISH"))
```
[the remark this answers is not in the document]
Comment (ashkriti) [reply Frontend/ui-customer/src/Flow.purs]: Done

**Review by witcher-shailesh — APPROVED** [review]

**Pull request merged**